admin/release.py

```diff
@@ -1,4 +1,4 @@
-# -*- test-case-name: admin.test.test_release -*-
+# -*- test-case-name: admin.test.test_release,admin.functional.test_release -*-  # noqa
```

Comment:
Fair point. I tested it in emacs, but this is also how `trial  --testmodule=admin/release.py` works. We use that feature for running our flocker.node.agent functional tests.